typing_extensions/LICENSE

```diff
@@ -0,0 +1 @@
+../LICENSE
```

Comment:
I take it this is a symbolic link. I'm not sure we're okay with those, though I'm not sure how to do it without one. I know that in typeshed we stopped using symbolic links because they caused problems on Windows, but of course that's a much larger project.

Reply:
Since the LICENSE will likely never change, I think it would be better just to copy it over.